spec/core/account/routes/getBeneficiaryBlocks.yml

```diff
@@ -0,0 +1,21 @@
+tags:
+  - Account routes
+summary: Get beneficiary blocks
+description: Gets an array of blocks where the account is the beneficiary.
+operationId: getBeneficiaryBlocks
+parameters:
+  - $ref: "../../../parameters/pageSize.yml"
+  - $ref: "../../../parameters/id.yml"
+  - $ref: "../../../parameters/ordering.yml"
+responses:
+  "200":
+    description: success
+    content:
+      application/json:
+        schema:
+          type: array
+          description: Array of blocks information.
+          items:
+            $ref: "../../block/schemas/BlockInfoDTO.yml"
```

Comment:
The output is slightly different, because it includes the id for paging.

Example:
```
[
    {
        "id": "5E3CD1498E18164DD5536133",
        "meta": { ... },
        "block": { ... }
    },
    ...
]
```